contracts/MerkleDistributor.sol

```diff
@@ -0,0 +1,115 @@
+pragma solidity 0.6.4;
+
+import "./interface/IBEP20.sol";
+import "./interface/ITokenHub.sol";
+import "./interface/IMerkleDistributor.sol";
+import "./lib/SafeMath.sol";
+import "./System.sol";
+import "./MerkleProof.sol";
+
+contract MerkleDistributor is IMerkleDistributor, System {
+    using SafeMath for uint256;
+
+    string public constant sourceChainID = "Binance-Chain-Tigris"; // TODO: replace with the real chain id
+    address public constant approvalAddress = 0xaAaAaAaaAaAaAaaAaAAAAAAAAaaaAaAaAaaAaaAa; // TODO: replace with the real address
```

Comment:
`approvalAddress` should be rotatable.

Reply:
yeah, may I confirm which user is allowed to rotate this?

Reply:
updated